src/content/resources/index.mdx

```diff
@@ -21,7 +21,20 @@ To see a comprehensive overview of OASIS+ Apparent Contractor Awardees across th
 | Resource | Description |
 | --------------------------| --------------------------|
 | [OASIS+ Systems, Tools, and Registration Sellers Guide [PDF, 9 pages]](/files/OASIS%2B_Systems%2C%20Tools%2C%20and%20Registration_Sellers%20Guide_%20Sep%202024%20(508).pdf) | A guide to the various registrations and tools required to manage your contract to help your company be in compliance with the OASIS+ program requirements. |
-| Training - proposal submission (coming soon) | Learn how to submit your proposal. |
+|[FedConnect](http://FedConnect.net) | For managing your OASIS+ contract |
+|  |   **FedConnect - [Ready, Set, Go!](https://www.fedconnect.net/FedConnect/Marketing/Documents/FedConnect_Ready_Set_Go.pdf) - FedConnect Registration Guide |
+|  |   **FedConnect - [OASIS+ Guide](/sellers-guide/make-changes-contract/) - Request contract changes |
+|[VSC Registration](https://vsc.gsa.gov/vsc/app-registration) | Vendor Support Center (VSC) for access to eBuy. |
+| |VSC Phone: [1-877-495-4849](1-877-495-4849) / VSC Email: vendor.support@gsa.gov |
+|GSA eBuy | For seeing OASIS+ order opportunities (after registration in the VSC)|
+|   |  **[eBuy Homepage](https://www.ebuy.gsa.gov/ebuy/) | 
+|   |  **[eBuy Seller Guide](https://www.ebuy.gsa.gov/ebuy/assets/content/eBuy-Contractor_jobaid.pdf) | 
```

Comment:
[suggest to specify file type and size with it](https://designsystem.digital.gov/components/link/). e.g. [eBuy Seller Guide [PDF, 22 pages]

Reply:
corrected